src/constants/index.js

```diff
@@ -1,7 +1,7 @@
-module.exports = {
-  AFRAME_CDN_ROOT: window.AFRAME_CDN_ROOT || 'https://cdn.aframe.io/',
-  AFRAME_INJECTED: 'aframe-injected',
-  DEFAULT_CAMERA_HEIGHT: 1.6,
-  DEFAULT_HANDEDNESS: 'right',
-  keyboardevent: require('./keyboardevent')
-};
+import { KEYCODE_TO_CODE } from './keyboardevent.js';
+
+export var AFRAME_CDN_ROOT = window.AFRAME_CDN_ROOT || 'https://cdn.aframe.io/';
+export var AFRAME_INJECTED = 'aframe-injected';
+export var DEFAULT_CAMERA_HEIGHT = 1.6;
+export var DEFAULT_HANDEDNESS = 'right';
+export var keyboardevent = KEYCODE_TO_CODE;
```

Comment:
Shouldn't it be 
```js
export var keyboardevent = {KEYCODE_TO_CODE}
```
here or
```js
import * as keyboardevent from './keyboardevent.js';
export var keyboardevent = keyboardevent;
```
?
We don't seem to import `keyboardevent` from `constants/index.js` anywhere, we always import it from `constants/keyboardevent.js`

We could also just remove the line.

Reply:
Ah, you're right, but I agree with just removing this line. We indeed only ever import directly from `constants/keyboardevent.js` it seems.